actors/common/src/main/java/org/sunbird/learner/actors/tac/UserTnCActor.java

```diff
@@ -82,6 +88,16 @@ private void acceptTNC(Request request) {
           ResponseCode.RESOURCE_NOT_FOUND.getResponseCode());
     }
 
+    // Check whether user account is a child account(passed in request) wth managedBy is empty
```

Comment:
use managedBy only. And should this be userId or requestUserId ?

Reply:
Done